connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -198,8 +198,7 @@ public Connector connector() {
     @Override
     public String toString() {
         return "WorkerConnector{" +
-                "connName='" + connName + '\'' +
-                ", connector=" + connector +
+                "id=" + connName +
```

Comment:
I'm actually struggling to find an example of what you're referring to -- I looked at a bunch of `toString` implementations in streams and most look either like this (presumably because IntelliJ generates them like this) or are clearly manually defined. Can you point to an example of what you're referring to?